docs/gallery/domain/brain_observatory.py

```diff
@@ -0,0 +1,187 @@
+"""
+Allen Brain Observatory
+=================================
+
+Create an nwb file from Allen Brain Observatory data.
+"""
+
+########################################
+# This example demostrates the basic functionality of several parts of the pynwb write API, centered around the optical physiology submodule (pynwb.ophys). We will follow and extract/transform/load (ETL) methodology, using the allensdk as a read API while leveraging the pynwb data model and write api to transform and write the data back to disk. # noqa: E501
+
+########################################
+# .. image:: http://interactive.blockdiag.com/image?compression=deflate&encoding=base64&src=eJyVkFEKAjEMRP89RS_gCQSvUrLd2W6wTWrTIiLe3S0KwoIf_k2SN5OQKWm4zEzRPQ7OyW1aOMEdz84a5566DV0qDNKoscrpN9aQS6IG2zEUrp2Nh3uUtYuwRG8FmHcoioZ1748QVEpDamla1ruxJo330eFMcaRtmwUfhAMlH1YSQdpllaoBZpvjzW5ZPuvc39N5WTxLQ10o4C_nOAPeEPP3Uc8XlV14ZQ # noqa: E501
+
+import datetime
+
+from allensdk.core.brain_observatory_cache import BrainObservatoryCache
+import allensdk.brain_observatory.stimulus_info as si
+
+from pynwb import NWBFile, NWBHDF5IO, TimeSeries
+from pynwb.ophys import OpticalChannel, DfOverF, ImageSegmentation
+from pynwb.image import ImageSeries, IndexSeries
+
+
+# Settings:
+ophys_experiment_id = 562095852
+save_file_name = 'basic_example.nwb'
```

Comment:
Can you change this name? Another tutorial already has this name, I think. In the event that someone wants to run the test suite and view the files, one of these would get clobbered.

Reply:
Aha, OK, will do.  I am also working on getting this thorugh some sort of sane testing, since allensdk is a dependency for getting this to work :/

Reply:
OK, this is done.